docs/index.rst

```diff
@@ -328,6 +329,21 @@ You need to specify
         }
     }
 
+Clone and archive git repository
```

Comment:
I have updated the server, you can check and amend the command :) and then I will merge it.

Reply:
👍